src/udev/memory_id/dmidecode.c

```diff
@@ -0,0 +1,777 @@
+/*
```

Comment:
Where is this code coming from? Is GPLv2+ the only license that we can use? LGPLv2+ would be preferable to match the rest of the code base. Either way, please add a SPDX header.

Reply:
The scary bits with the hex numbers are from dmidecode, so GPLv2+.